deltacat/__init__.py

```diff
@@ -93,6 +94,7 @@
     "default_namespace",
     "write_to_table",
     "read_table",
+    "create_scan_plan",
```

Comment:
Can we move this from `deltacat/__init__.py` to `deltacat/storage/__init__.py` (the average DeltaCAT end-user shouldn't need to invoke `create_scan_plan` directly).